spark/src/test/scala/org/apache/comet/CometArrayExpressionSuite.scala

```diff
@@ -218,7 +218,7 @@ class CometArrayExpressionSuite extends CometTestBase with AdaptiveSparkPlanHelp
     }
   }
 
-  test("array_contains") {
+  test("array_contains - int values") {
```

Comment:
I would comment that INTs are in separate tests as ints require incompatible flag